modules/guardrails-orchestrator-hap-scenario.adoc

```diff
@@ -13,16 +13,16 @@ The following example demonstrates how to use Guardrails Orchestrator to monitor
 * You have downloaded and installed the OpenShift command-line interface (CLI). See link:https://docs.redhat.com/en/documentation/openshift_container_platform/{ocp-latest-version}/html/cli_tools/openshift-cli-oc#installing-openshift-cli[Installing the OpenShift CLI^].
 
 ifdef::upstream[]
-* You have deployed the Guardrails Orchestrator and related detectors. See link:{odhdocshome}/monitoring_data_science_models/#deploying-the-guardrails-orchestrator-service_monitor[Deploying the Guardrails Orchestrator].
+* You have deployed the Guardrails Orchestrator and related detectors. For more information, see link:{odhdocshome}/monitoring_data_science_models/#deploying-the-guardrails-orchestrator-service_monitor[Deploying the Guardrails Orchestrator].
```

Comment:
@syaseen-rh heads up, I fixed a broken link that was merged in another PR here, so this is a technically new file for this PR: https://github.com/opendatahub-io/opendatahub-documentation/pull/708/files#diff-ceeb06c2cc384aa4d2fb9ec167be5b7f1c4a6ada0bd6c0bc3ef5c3d904346ba3R16-R26

I had some git weirdness in the process of rebasing but I think I've ironed it out now.